styles/config/vocabularies/Docs/accept.txt

```diff
@@ -47,6 +47,7 @@ Atlassian
 autogenerate
 autogenerated
 [Au]toscal[e|er|ing]
```

Comment:
```suggestion
[Aa]utoscal[e|er|ing]
```

Reply:
I saw that entry, but for some reason that didn't work when I tried earlier: 
- https://app.circleci.com/pipelines/github/circleci/circleci-docs/57898/workflows/cd0df763-3628-4ef5-95cc-0b9fc031199c/jobs/175472?invite=true#step-102-3436_87
- https://github.com/circleci/circleci-docs/commit/ddddb153ae175298dd6eced8e83d2ccc62970b4d#diff-3f79e7801d962d37839202599856add07a65e60368d3aa169475bee201f5c011